### PR TITLE
mysql tls support

### PR DESCRIPTION
**Pull request by Lagovas**

### CODE REVIEW

File: cmd/acraserver/acraserver.go [hunk 1]
```diff
@@ -169,14 +178,23 @@ func main() {
...
+	var tlsConfig *tls.Config
+	if *useTls || *tlsKey != "" {
+		tlsConfig, err = network.NewTLSConfig(*tlsSNI, *tlsCA, *tlsKey, *tlsCert)
 		if err != nil {
 			log.WithError(err).WithField(logging.FieldKeyEventCode, logging.EventCodeErrorTransportConfiguration).
 				Errorln("Configuration error: can't get config for TLS")
 			os.Exit(1)
 		}
+		// need for testing with mysql docker container that always generate new certificates
+		if Test == TEST_MODE {
```
Comment (vixentael) [inline cmd/acraserver/acraserver.go]: Won't it be better to rename `Test` into `SkipVerifyingCertificated`?

Comment (Lagovas) [reply cmd/acraserver/acraserver.go]: thought to name it more common to use in future tests if need

Comment (vixentael) [reply cmd/acraserver/acraserver.go]: what about 'IsRunningTest'? 'TestOnly'?

Comment (vixentael) [reply cmd/acraserver/acraserver.go]: by the way, why do you check 
`if Test == TEST_MODE`

instead of `if Test == True`?

Comment (Lagovas) [reply cmd/acraserver/acraserver.go]: on compile time we can change only string values

Comment (Lagovas) [reply cmd/acraserver/acraserver.go]: will rename to TestOnly

**Pull request merged**